example/use_cases/employer_university_flow/employer_university_flow/wallet.go

```diff
@@ -0,0 +1,126 @@
+package employer_university_flow
+
+import (
+	gocrypto "crypto"
+	"errors"
+	"fmt"
+	"sync"
+
+	"github.com/TBD54566975/ssi-sdk/credential"
+	"github.com/TBD54566975/ssi-sdk/crypto"
+	"github.com/TBD54566975/ssi-sdk/did"
+)
+
+// A sample wallet
+// This would NOT be how it would be stored in production
+// But serves for demonstrative purposes
+// This holds the assigned dids
+// private keys
+// and vCs
+type SimpleWallet struct {
+	vCs  map[string]*credential.VerifiableCredential
+	keys map[string]gocrypto.PrivateKey
+	dids map[string]string
+	mux  *sync.Mutex
+}
+
+func NewSimpleWallet() *SimpleWallet {
```

Comment:
I like this wallet, perhaps move it up so it benefits all use cases?

Reply:
👀 yea! sounds good

Reply:
done.